src/alchemlyb/estimators/bar_.py

```diff
@@ -65,29 +89,47 @@ def __init__(
         maximum_iterations=10000,
         relative_tolerance=1.0e-7,
         method="false-position",
+        n_bootstraps=0,
         verbose=False,
     ):
 
         self.maximum_iterations = maximum_iterations
         self.relative_tolerance = relative_tolerance
         self.method = method
+        self.n_bootstraps = n_bootstraps
         self.verbose = verbose
 
         # handle for pymbar.BAR object
         self._bar = None
 
-    def fit(self, u_nk):
+    def fit(self, u_nk, use_mbar=False, compute_entropy_enthalpy=False):
         """
         Compute overlap matrix of reduced potentials using
         Bennett acceptance ratio.
 
+        Sets the attributes: delta_f_ and d_delta_f_
+
         Parameters
         ----------
         u_nk : DataFrame
             u_nk[n,k] is the reduced potential energy of uncorrelated
             configuration n evaluated at state k.
 
+        use_mbar : bool, optional, default=False
```

Comment:
Sorry do you mind explain to me why do you want to do this in BAR? I think if you want the enthalpic and entropic contributions, you would just run MBAR?